crud/student-crud.js

```diff
@@ -0,0 +1,21 @@
+const studentModel = require(__dirname + "/../models/student.js");
+
+class studentCrud {
+  constructor(connection, dataTypes) {
+    this.connection = connection;
+    this.dataTypes = dataTypes;
+    this.student = studentModel(connection, dataTypes);
+  }
+
+  async createStudent(requestBody) {
+    try {
+      return await this.connection.transaction(async (t) => {
+        return await this.student.create(requestBody, { transaction: t });
+      });
+    } catch (error) {
+      return error;
+    }
+  }
+}
+
+module.exports = studentCrud;
```

Comment:
We can remove the class structure and instead have individual CRUD functions that we can export. We import db object from models/index.js that will have connection and Student model. We can import DataTypes from 'sequelize'.

Reply:
Changes have been made.